packages/lib/src/vis/shared/Html.tsx

```diff
@@ -93,11 +95,7 @@ function Html(props: Props) {
     }
 
     return () => {
-      if (parentElement) {
-        el.remove();
-      }
-
-      ReactDOM.unmountComponentAtNode(el);
+      el.remove();
```

Comment:
This code initially came [from drei](https://github.com/pmndrs/drei/blob/master/src/web/Html.tsx#L174), but doing the unmount in the clean-up of this effect was problematic for me in `FloatingControl`, where the `floatingToolbar` is a dependency of the `useLayoutEffect`. When `floatingToolbar` becomes defined and everything re-renders, the effect clean-up function gets run, which unmounts the control from the DOM. Since the effect that mounts the control doesn't run again during this re-render, the control remains unmounted...

This works in drei, because the ref is set on initial mount, not on subsequent re-render. Anyway, so I moved the unmount in the clean-up function of the `useEffect` that does the mount... Seems more logical.

Note also that `if (parentElement)` is unnecessary, since calling `remove()` on an already-removed element has no effect.